src/main/java/net/neoforged/neoforge/common/EffectCures.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.common;
+
+import java.util.Set;
+
+public class EffectCures {
+    public static final EffectCure MILK = EffectCure.get("milk");
```

Comment:
These could use some javadocs briefly describing how they work. Mostly modders might wonder whether they should use milk,  protected by totem, or a custom effect cure for their effect that cures something, and might not be aware that honey cures poison.

Reply:
I added some simple docs, I currently can't think of anything better, so I'm open to suggestions